### PR TITLE
Improve Malyan M200 integration

### PR DESCRIPTION
**Pull request by me-no-dev**
### Description

This pull request intends to improve the support for Malyan M200 in Marlin.
I have tested those changes fully for a couple of weeks and 20+ prints.

### Benefits

- It brings support for all LCD functions
- Also supports all web commands, except upload to the SD card (not enough info on the protocol, but have muted it to not hang the printer)
- Adds some comments to the default M200 config, so users can easily configure the 2 available FANs
- Adds good PID values for stock machines
- Fixes an issue caused by the LCD sending command to turn off heaters while heating or printing

### Related Issues

Not aware of any open issues about any of the changes. Did not find much on the topic anyway

### CODE REVIEW

File: Marlin/src/lcd/extui_malyan_lcd.cpp [hunk 1]
```diff
@@ -350,6 +369,47 @@ void process_lcd_command(const char* command) {
...
+  static char inbound_buffer[MAX_CURLY_COMMAND];
+
+  if (!parsing_lcd_cmd && b == '{') {
+    if(inbound_count){
+      inbound_buffer[inbound_count] = '\0';
+      queue.enqueue_one_now(inbound_buffer);
+    }
+    parsing_lcd_cmd = true;
+    inbound_count = 0;
+    inbound_buffer[0] = 0;
```
Comment (thinkyhead) [inline Marlin/src/lcd/extui_malyan_lcd.cpp]: It looks like this should actually be…
```cpp
inbound_count = 1;
inbound_buffer[0] = '{';
```

**Comment by thinkyhead** [comment]
I made some tweaks to `parse_lcd_byte` so please confirm nothing is broken by this.

**Comment by thinkyhead** [comment]
Rebased and squashed. To update your working copy use the Git Console and type…
```
git fetch origin
git checkout malyan-m200
git reset --hard origin/malyan-m200
```

**Comment by me-no-dev** [comment]
thanks for the fast merge :) I checked the changes to `parse_lcd_byte` and in my head I got the correct results. Will give it a go later on the machine.

**Pull request merged**